src/amo/components/SearchResult/styles.scss

```diff
@@ -122,6 +122,7 @@ $icon-default-size: 32px;
   flex-grow: 1;
   flex-wrap: wrap;
   font-size: $font-size-default;
+  gap: 6px;
```

Comment:
🤯 did not know something like this exists! thanks Bob!

Reply:
> 🤯 did not know something like this exists! thanks Bob!

Neither did I!